chain-signatures/node/src/protocol/triple.rs

```diff
@@ -144,19 +144,11 @@ impl TripleManager {
         }
     }
 
-    pub async fn insert(&mut self, triple: Triple) {
-        tracing::debug!(id = triple.id, "inserting triple");
+    pub async fn insert(&mut self, triple: Triple, mine: bool) {
+        tracing::debug!(id = triple.id, mine, "inserting triple");
         self.gc.remove(&triple.id);
```

Comment:
Let's follow the same pattern when removing things from gc. It is not consistent with how it's done in presignatures.